src/octree/mod.rs

```diff
@@ -397,3 +380,54 @@ fn maybe_push_node(
         });
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct Frustum<S: BaseFloat> {
+    pub matrix: Matrix4<S>,
```

Comment:
Could you remind me why this contains the second matrix? I forgot. Also, the fields probably do not need to be public.

Reply:
i moved it in math, the matrix is used by the contains function

// TODO(ksavinash9) update after https://github.com/rustgd/collision-rs/issues/101 is resolved.
pub fn contains<S: BaseFloat>(projection_matrix: &Matrix4<S>, point: &Point3<S>) -> bool {

the issue in collision is not resolved yet